test/JDBC/upgrade/latest/schedule

```diff
@@ -133,6 +133,7 @@ babel_datetime
 Babel_domain_mapping_test
 BABEL-EXECUTE_AS_CALLER
 BABEL-EXTENDEDPROPERTY
+BABEL-EXTENDEDPROPERTY-v2
```

Comment:
We should add this to older version's schedule files as well to cover other m\Mvu workflows. @2jungkook @rishabhtanwar29 can comment better.

Reply:
Yep, since `prepare` script is using newly added procedures inside test procedures, it could be executed on older versions without any issue. So we should add this test all previous upgrade scripts.

Reply:
done

Reply:
This should be happening for the versions for which sp_xxxxextendedproperty is not supported.

Reply:
Ahh, generally we doesn't check the objects inside a procedure's body at `CREATE` time but reason for the above errors is that we are detecting unsupported procedure's in ANTLR parser itself [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/a7e79421f66030d1d3b97cf09136da4a4fce8958/contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp#L1490). I was not aware of this before so looks like we won't be able to add this test in previous versions since it looks like all these procedures are new.